tests/e2e/testdata/scenarios/direct/llm_immutable_field/description/SCENARIO_README.MD

```diff
@@ -0,0 +1,3 @@
+Step1: Create a logginglogmetric resource, operation succeeds
+Step2: Update spec.description field for the resource, operation succeeds. We'll see PUT request with updated 
```

Comment:
I don't think we have a scenario test where an update is blocked by the immutability webhook (unless I'm missing something)?

Reply:
IIUC, the `description` is mutable, so this path is valid. Are you looking for the label field (immutable, fails without PUT request)  
https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/1750/files#diff-168ebeca2eba2ea5385e8de5887d460b5bd7726f884323e116dc82ca4a43b113

Reply:
oh, I see the problem. @gemmahou I think we want to add the immutability check for `metricDescriptor.description` (output only, immutable) rather than the `description` field.

Reply:
`spec.description` is mutable, I have this test here to make sure mutable fields can be updated.

`spec.metricDescriptor.labels.description` field is immutable, however, with the current DCL controller, we do not have webhook or controller level checks for labels field, instead, GCP API will failed if any existing labels fields(description, key, valueType) are changed. That's why I have a separate test for labels: https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/7164701044d5bd747540aed65c3603564031ed6a/tests/e2e/testdata/scenarios/direct/llm_immutable_field/labels/SCENARIO_README.MD

I guess the reason that we do not have checks on `spec.metricDescriptor.labels` field is, it's not allowed to modify existing labels, but it's allowed to add a new label with description, key and valueType(deletion is not allowed as well if i remember correctly). So It's a bit tricky for us to handle it in our webhook or controller.

Reply:
Can we add another test to make sure updating the `spec.metricDescriptor.labels.description` will fail? 
We don't need specific test for `spec.description` .